### PR TITLE
added function to sort articles by date, title, comments & votes count also handle errors

### PR DESCRIPTION
**Pull request by MariusVen**

### CODE REVIEW

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ export default function ArticleCard({ article }) {
       </Link>
       <br></br>
       <div className="comment_count">
+        {article.created_at.slice(0, 10)}
```
Comment (emilyb93) [inline src/components/ArticleCard.jsx]: Make sure you are putting any kind of text in a tag that represents it for accessibility such as <p>

File: src/components/ArticleCard.jsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ export default function ArticleCard({ article }) {
...
+        <br></br>
         <b>{article.comment_count}</b> comments <b>{article.votes}</b> votes
```
Comment (emilyb93) [inline src/components/ArticleCard.jsx:12]: Same here for the comments and votes text

**Review by emilyb93 — APPROVED** [review]
Just make sure you add the tags for the bits i've pointed out and this is fine to go :)

File: src/components/ArticleCard.jsx (hunk 1 above)
Comment (MariusVen) [reply src/components/ArticleCard.jsx]: Just fixed. Thank you

**Pull request merged**